Build/libHttpClient.import.props

```diff
@@ -65,7 +65,10 @@
   </ItemDefinitionGroup>
 
   <!--Bring in XCurl on GDK-->
-  <ImportGroup Condition="'$(HCPlatform)' == 'GDK' AND '$(GameDK)' != ''">
+  <ImportGroup Condition="'$(HCPlatform)' == 'GDK' AND '$(GameDK)' != '' AND '$(GDKVer)' == '251000' ">
```

Comment:
this won't work in builds later that 25100 cause it defaults to old paths.  
just keep it simple and do

  <ImportGroup Condition="'$(HCPlatform)' == 'GDK' AND '$(GameDK)' != ''">
<Import Project="$(GameDK)\$(XdkEditionTarget)\GRDK\ExtensionLibraries\Xbox.XCurl.API\ExtensionLibrary.props"/>
</ImportGroup>

Reply:
Using "exists" and checking for the specific file instead